sklearn/multiclass.py

```diff
@@ -523,7 +524,29 @@ def predict(self, X):
         y : numpy array of shape [n_samples]
             Predicted multi-class targets.
         """
+        Y = self.decision_function(X)
+        return self.classes_[Y.argmax(axis=1)]
+
+    def decision_function(self, X):
+        """Returns the decision values (votes + scaled aggregated scores) of 
+        the samples.
+
+        The scaled scores are added in order to disambiguate between the
```

Comment:
I think "scores" is a bad name here as it can be confused with sklearn scoring metrics (accuracy, f1-score...).

Instead I would say the "the normalized sum of pair-wise classification confidence levels". In the code I would replace the local variables named `score` by `confidence`.